_partials/_timescaledb_supported_linux.md

```diff
@@ -0,0 +1,11 @@
+
+| Operation system                | Version                                                               |
```

Comment:
@svenklemm can you please double check the distros?

Reply:
We dont build Red Hat 7 anymore